scripts/bluepill.sh

```diff
@@ -57,9 +57,25 @@ test_runtime()
   }
 }
 
+simulator_cleanup()
+{
+echo "Clean up simulators"
+xcrun simctl list | grep BP | sed 's/).*$//g;s/^.*(//g;' | while read x; do xcrun simctl shutdown $x >/dev/null; xcrun simctl delete $x >/dev/null; done
+}
+
 bluepill_build()
 {
   set -o pipefail
+  xcodebuild \
+    -project Bluepill-cli/Bluepill-cli.xcodeproj \
+    -scheme bluepill-cli \
```

Comment:
you can define bluepill-cli to build bluelib as a dependency first instead of calling two separate build commands

Reply:
bp-cli is not importing headers from BluepillLib, it will cause duplicate definition errors if we set the dependency.

Reply:
I don't think you need to import headers to set dependencies... can' t you just add it in Xcode's dependencies in the scheme?